scenarios/sites/setting/environments.ftd

```diff
@@ -0,0 +1,11 @@
+-- import: ui.fifthtry.com/pages/sites/setting/environments
+-- import: ui.fifthtry.com/scenarios/john-doe
+-- import: ui.fifthtry.com/records/sites as sites-records
+-- import: ui.fifthtry.com/scenarios/sites/site-info
+-- import: ui.fifthtry.com/scenarios/scenario-site
+
+-- environments.page:
+site-data: scenario-site
+user-data: $john-doe.jd
+site: $site-info.acme-site
+dashboard-url: /scenarios/user/dashboard/
```

Comment:
You have to pass dummy data here.

Reply:
fixed in https://github.com/FifthTry/ui/pull/58/commits/2b5817741bbc6c82c99e51f4fc088e2fff02ce3e